app/src/main/java/org/sopt/melon/data/repositoryimpl/AlbumRepositoryImpl.kt

```diff
@@ -0,0 +1,17 @@
+package org.sopt.melon.data.repositoryimpl
+
+import org.sopt.melon.data.mapper.toForYouAlbumModel
+import org.sopt.melon.data.model.ForYouAlbum
+import org.sopt.melon.data.remote.datasource.AlbumDataSource
+import org.sopt.melon.data.repository.AlbumRepository
+import javax.inject.Inject
+
+class AlbumRepositoryImpl @Inject constructor(
+    private val albumDataSource: AlbumDataSource,
+) : AlbumRepository {
+    override suspend fun getAlbum(albumId: Int): Result<ForYouAlbum> =
+        runCatching {
+            val response = albumDataSource.getAlbum(albumId)
+            response.result.toForYouAlbumModel()
+        }
```

Comment:
```suggestion
        runCatching {
            albumDataSource.getAlbum(albumId).result.toForYouAlbumModel()
        }
```
이렇게 한 줄 처리 하지 않고 나눈 이유는
라인이 너무 길어져서..?

Reply:
p3: 네 ㅎ 명시적으로 response 를 받아오는 것을 구분하고자 이렇게 작성했었습니다